test/mempool-test.js

```diff
@@ -440,6 +497,8 @@ describe('Mempool', function() {
       memory: true
     });
 
+    const wallet = new MemWallet();
```

Comment:
Required? There is a global `wallet` at the top of the test. Also, we don't need to set network?

Reply:
It's "cleaner" with new instance.